android/app/src/main/res/values-es/strings.xml

```diff
@@ -1187,37 +1187,37 @@
 	<string name="type.highway.footway.demanding_mountain_hiking">Camino</string>
 	<string name="type.highway.footway.difficult_alpine_hiking">Camino</string>
 	<string name="type.highway.footway.hiking">Sendero</string>
-	<string name="type.highway.footway.mountain_hiking">Camino</string>
+	<string name="type.highway.footway.mountain_hiking">Sendero de montaña</string>
 	<!-- These translations are used for all type.highway.*.tunnel. -->
 	<string name="type.highway.footway.tunnel">Túnel peatonal</string>
 	<string name="type.highway.ford">Vado</string>
-	<string name="type.highway.living_street">Calle</string>
+	<string name="type.highway.living_street">Calle de prioridad peatonal</string>
 	<!-- These translations are used for all type.highway.*.bridge. -->
 	<string name="type.highway.living_street.bridge">Puente</string>
 	<!-- These translations are used for all type.highway.*.tunnel. -->
 	<string name="type.highway.living_street.tunnel">Túnel</string>
-	<string name="type.highway.motorway">Calle</string>
+	<string name="type.highway.motorway">Autopista</string>
 	<!-- These translations are used for all type.highway.*.bridge. -->
-	<string name="type.highway.motorway.bridge">Puente</string>
+	<string name="type.highway.motorway.bridge">Puente de autopista</string>
 	<!-- These translations are used for all type.highway.*.tunnel. -->
-	<string name="type.highway.motorway.tunnel">Túnel</string>
+	<string name="type.highway.motorway.tunnel">Túnel de autopista</string>
 	<string name="type.highway.motorway_junction">Salida</string>
-	<string name="type.highway.motorway_link">Calle</string>
+	<string name="type.highway.motorway_link">Enlace de autopista</string>
 	<!-- These translations are used for all type.highway.*.bridge. -->
 	<string name="type.highway.motorway_link.bridge">Puente</string>
 	<!-- These translations are used for all type.highway.*.tunnel. -->
 	<string name="type.highway.motorway_link.tunnel">Túnel</string>
 	<string name="type.highway.path">Camino</string>
-	<string name="type.highway.path.alpine_hiking">Camino</string>
-	<string name="type.highway.path.bicycle">Camino</string>
+	<string name="type.highway.path.alpine_hiking">Senderismo alpino</string>
+	<string name="type.highway.path.bicycle">Camino para bicicletas</string>
 	<!-- These translations are used for all type.highway.*.bridge. -->
 	<string name="type.highway.path.bridge">Puente</string>
-	<string name="type.highway.path.demanding_alpine_hiking">Camino</string>
-	<string name="type.highway.path.demanding_mountain_hiking">Camino</string>
+	<string name="type.highway.path.demanding_alpine_hiking">Senderismo alpino exigente</string>
```

Comment:
"Senderismo" is the sport or activity. "Sendero" is the path.

Reply:
Changed to "Camino de senderismo alpino exigente".

Please check [PR 7558](https://github.com/organicmaps/organicmaps/pull/7558).